src/thriveopedia/fossilisation/FossilisationButton.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Godot;
+
+/// <summary>
+///   Button shown above organisms in pause mode to fossilise (save) them.
+/// </summary>
+public class FossilisationButton : TextureButton
+{
+    [Export]
+    public Texture AlreadyFossilisedTexture = null!;
+
+    /// <summary>
+    ///   The microbe this button is attached to.
+    /// </summary>
+    public Microbe AttachedMicrobe = null!;
```

Comment:
From looking at things, it should be possible to change this to `Spatial` as this doesn't seem to need to know anything about the species itself.